tests/test_filtsmooth/test_gaussfiltsmooth/test_kalman.py

```diff
@@ -121,16 +121,16 @@ def test_predict_value(self):
             / (2 * self.lam)
             * (1 - scipy.linalg.expm(2 * self.drift * self.delta_t))
         )
-        expectedmean = np.squeeze(ah @ (self.initrv.mean() * np.ones(1)))
-        expectedcov = np.squeeze(ah @ (self.initrv.cov() * np.eye(1)) @ ah.T + qh)
-        self.assertApproxEqual(expectedmean, pred.mean())
-        self.assertApproxEqual(expectedcov, pred.cov())
+        expectedmean = np.squeeze(ah @ (self.initrv.mean * np.ones(1)))
+        expectedcov = np.squeeze(ah @ (self.initrv.cov * np.eye(1)) @ ah.T + qh)
+        self.assertApproxEqual(expectedmean, pred.mean)
+        self.assertApproxEqual(expectedcov, pred.cov)
 
     def test_update(self):
-        data = np.array([self.measmod.sample(0.0, self.initrv.mean() * np.ones(1))])
+        data = np.array([self.measmod.sample(0.0, self.initrv.mean * np.ones(1))])
```

Comment:
Fix für den test: Das äußere `np.array` macht `data` hier zu einer Matrix, und damit wird in `Kalman` auch irgendwas matrix-valued. Ich glaub das `np.array` kann einfach weg, ich seh jetzt keinen Grund wozu man das bräuchte.

Die Zeile wäre dann: 
```python
        data = self.measmod.sample(0.0, self.initrv.mean * np.ones(1))
```
Damit sollten die tests laufen :)